apps/examples/smart_test/smart_test.c

```diff
@@ -590,11 +590,12 @@ int main(int argc, FAR char *argv[])
 int smart_test_main(int argc, char *argv[])
 #endif
 {
-	int ret, opt;
+	int ret;
+	int opt;
 
 	/* Argument given? */
 
-	optind = -1;
+	int optind = -1;
```

Comment:
This is a global variable which is defined at other file so that you must not add `int`.

Reply:
i second Mr Sunghan opinion